apache/ratis
# RATIS-2083. Timer thread leak when log level is DEBUG

## What changes were proposed in this pull request?

During a DN issue investigation, it's found that there are thousands of Timer threads in the jstack output. A further investigation shows that Ratis creates new Timer() in every SlidingWindow#RequestMap object when log level is debug. See the attached jstack file in JIRA.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/RATIS-2083

## How was this patch tested?
Manual test. 
1.  enable Ozone DEBUG log level
2. Start a Ozone docker bases cluster
3. create bucket, put multiple files into this bucket
4. get jstack of DN process
5. check there is only one "Timer-SlidingWindow" timer thread.  Before this patch, for each new file put into bucket, there will be a new Timer thread created.
